ArgusCore/src/main/java/com/salesforce/dva/argus/entity/Policy.java

```diff
@@ -0,0 +1,276 @@
+    private String timeUnit;
+    
+	@Basic(optional = false)
+    @Column(name = "default_value",nullable = false)
+    private double defaultValue;
+    
+	@Basic(optional = false)
+    @Column(name = "cron_entry", nullable = false)
+    private String cronEntry;
+    
+    @OneToMany(mappedBy="policy", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<SuspensionLevel> suspensionLevelList;
+    
+    @OneToMany(mappedBy="policy", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Infraction> infractionList;
+    
+  //~ Constructors *********************************************************************************************************************************
+
+	/**
+     * Creates a new Policy object.
+     *
+     * @param  creator      The creator of this policy.
+     * @param  service      The service for this policy. Cannot be null.
+     * @param  name     	The name for this policy. Cannot be null.
+     * @param  owner 		The owner for this policy. Cannot be null.
+     * @param  user  		The user for this policy. Cannot be null.
+     * @param  subSystem  	The subSystem for this policy. Cannot be null.
+     * @param  metricName  	The metric name for this policy. Cannot be null.
+     * @param  triggerType  The trigger type for this policy. Cannot be null.
+     * @param  aggregator  	The aggregator for this policy. Cannot be null.
+     * @param  threshold  	The threshold for this policy. Cannot be null.
+     * @param  timeUnit  	The time unit for this policy. Cannot be null.
+     * @param  defaultValue The default value for this policy. Cannot be null.
+     * @param  cronEntry  	The cron entry for this policy. Cannot be null.
+     */
+    public Policy(PrincipalUser creator, String service, String name, List<String> owner, List<String> user, String subSystem,
+			String metricName, TriggerType triggerType, String aggregator, List<Double> threshold, String timeUnit,
+			double defaultValue, String cronEntry) {
+		super(creator);
+		setService(service);
+		setName(name);
+		setOwner(owner);
+		setUser(user);
+		setSubSystem(subSystem);
+		setMetricName(metricName);
+		setTriggerType(triggerType);
+		setAggregator(aggregator);
+		setThreshold(threshold);
+		setTimeUnit(timeUnit);
+		setDefaultValue(defaultValue);
+		setCronEntry(cronEntry);
+	}
+    
+    /** Creates a new Policy object. */
+    protected Policy() {
+        super(null);
+    }
+  //~ Methods **************************************************************************************************************************************
+    /**
+     * Finds a policy given its name.
+     *
+     * @param   em      The entity manager to use. Cannot be null.
+     * @param   name  	The name of the policy. Cannot be null or empty.
+     *
+     * @return  The corresponding policy or null if no policy having the specified name exists for this name.
+     */
+    public static Policy findByName(EntityManager em, String name) {
```

Comment:
this should be called findByNameAndService, It should also take the service name as a parameter.  The named query requires both.  Be sure to add query.setParameter("service", service);